<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgVisorQR</class>
 <widget class="QDialog" name="dlgVisorQR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visor de QR</string>
  </property>
  <widget class="QPushButton" name="btnAbrir">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>40</y>
     <width>61</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblArchivo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona Archivo de Imagen a abrir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtArchivo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>861</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>911</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblImagen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>921</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
